### PR TITLE
Improve grammar in readme

### PR DESCRIPTION
**Pull request by mehagar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,20 +42,20 @@ On all platforms run the following in the VireoSDK folder to install JavaScript-
 - emsdk toolchain (see following)
 
 ### Installing the emsdk toolchain
-The emsdk toolchain is used to cross-compile Vireo to Vireo.js to run in Node.js and Browser environments.
+emscripten (emsdk) is used to cross-compile Vireo to Vireo.js to run in Node.js and web browser environments.
```
Comment (rajsite) [inline README.md]: Emscripten is a set of command line tools used to cross-compile...  The emsdk tool is used to install binaries of the emscripten command line tools and its required dependencies.

File: README.md [hunk 2]
```diff
@@ -42,20 +42,20 @@ On all platforms run the following in the VireoSDK folder to install JavaScript-
...
 
-1. Create a directory outside of the VireoSDK folder to store the emsdk toolchain binaries
+1. Create a directory outside of the VireoSDK folder to store the emsdk toolchain binaries.
 
 2. Do one of the following:
```
Comment (rajsite) [inline README.md]: If you want to remove the or then emphasize the one. `**ONE**`

File: README.md (hunk 1 above)
Comment (mehagar) [reply README.md]: You're right. I reverted this.

File: README.md (hunk 2 above)
Comment (mehagar) [reply README.md]: Good idea.

**Pull request merged**